.travis.yml

```diff
@@ -17,6 +17,7 @@ before_install:
 install:
   - docker build -t savaslabs/squabble:latest .
```

Comment:
Yeah! Actually just learned about that option and have implemented in on the branch for dockerized deployment.